src/packages.builds

```diff
@@ -16,6 +16,7 @@
     <Project Include="*\pkg\**\*.pkgproj" Condition="'$(BuildAllConfigurations)' == 'true' OR '$(DotNetBuildFromSource)' == 'true'">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+    <!-- DO NOT add OOB package authoring changes here. Use the next ItemGroup. -->
```

Comment:
@joperezr @ericstj I'm adding this comment to prevent accidental addition of OOB package changes here.